proto/message.py

```diff
@@ -375,7 +375,7 @@ def from_json(cls, payload, *, ignore_unknown_fields=False) -> "Message":
         return instance
 
     def to_dict(
-        cls, instance, *, use_integers_for_enums=True, preserving_proto_field_name=True
+        cls, instance, *, use_integers_for_enums=True, preserving_proto_field_name=True, including_default_value_fields=True,
```

Comment:
Please wrap:
```suggestion
        cls,
        instance,
        *,
        use_integers_for_enums=True,
        preserving_proto_field_name=True,
        including_default_value_fields=True,
```

Also, please add `including_default_value_fields` to the docstring below.